smart_open/s3.py

```diff
@@ -553,6 +573,24 @@ def __exit__(self, exc_type, exc_val, exc_tb):
         else:
             self.close()
 
+    def __str__(self):
+        return "smart_open.s3.BufferedOutputBase(%r, %r)" % (
+            self._object.bucket_name,
+            self._object.key
```

Comment:
```suggestion
            self._object.key,
```

But really, I think the above should fit on one line and still be within limits.